Bottom: 1a9a5cf5a4a37f248d4823754e359070b8f02473
Top:    31656d0ca72c03f7778c9add1f9519f7eea21a65
Author: genius user <genius@ReliableEmbeddedSystems.com>
Date:   2019-06-17 15:41:34 +0000

LAYERSERIES_COMPAT for bsp tool

Signed-off-by: genius user <genius@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index 195dbf6eff..b18c480656 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "thud"
+LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus"
